problems/objects.js

```diff
@@ -50,29 +60,40 @@ const eveAppleSet = () => {};
  *
  */
 
-const appleSet = () => {};
-
+const appleSet = (obj, name, num) => {
+  obj[name] = num
+  return obj;
+};
 /**
  * Takes in an object with peoples names as the keys and
  * the number of apples they have as values.
  * Should return the total number of apples for Adam and Eve.
  * @param {Object} appleCountByName - apple object with names and numbers
  * @returns {number} Sum of Adam and Eve's apples.
- *
+ * 
  */
 
-const adamAndEveApples = () => {};
+const adamAndEveApples = (obj) => {
+  let total = obj["Adam"] + obj["Eve"]
```

Comment:
Can we refactor this code to be one line? Do we need the variable 'total' in order to return the number?

Reply:
Hello Samantha 
Thank you for you comment. we dont the variable total we can return obj = obj["Adam or if i know the index" + obj["Eve" or index number] 
};